vetur/tags.json

```diff
@@ -1,10 +1,10 @@
 {
   "router-view": {
     "attributes": ["name", "route"],
-    "description": "Component that renders the matched component for the current location. Components rendered by `<router-view>` can also contain their own `<router-view>` to render nested routes."
+    "description": "Component that renders the matched component for the current location. Components rendered by `<router-view>` can also contain their own `<router-view>` to render nested routes.\n\n[API Reference](https://router.vuejs.org/api/#router-link)"
   },
   "router-link": {
     "attributes": ["to", "activeClass","exactActiveClass", "custom"],
```

Comment:
Why are you not including all attributes for `router-link`?

Reply:
I forgot `ariaCurrentValue` because I added it later I think

Reply:
all the other props have been removed

Reply:
🤦‍♂️ I was comparing it against old Vue router docs. Do you have a new one available? Can't find it. Also if they would be in different sites I should link to the one for vue-router-next.
(BTW I wasn't aware, are all attributes renamed to camelCase?)

Reply:
There isn't an official link yet, so I think we can add that later.
props can be passed both ways but by what you are saying I understand I should provide the kebab-case version for autocompletion 😄

Reply:
@posva I can add functionality to handle camelCase to kebab-case as well. No worries, just need to do a bit of changes in Vetur side.

Reply:
I think autocompleting on kebab-case is the way to go because it works on different contexts whereas camelCase doesn't. We should just use kebab-case in these files